src/Protocol/ActivityPub/Processor.php

```diff
@@ -541,7 +541,14 @@ public static function followUser($activity)
 		// Ensure that the contact has got the right network type
 		self::switchContact($item['author-id']);
 
-		Contact::addRelationship($owner, $contact, $item, '', false, $note);
+		$result = Contact::addRelationship($owner, $contact, $item, false, $note);
+		if ($result === false) {
+			ActivityPub\Transmitter::sendContactReject($item['author-link'], $item['author-id'], $owner['uid']);
```

Comment:
I don't like sending a contact reject in case of a blocked account. The other person doesn't need to know if we just ignore the person or have blocked the person.

Reply:
I think they do, because there is a difference between ignore and block. Do we have a way to ignore a contact without following them/them following us?

Reply:
I removed the follow reject response until we sort this out.

Reply:
We can just hit the "ignore" button on the introduction they send us and they will never get a response (as long as you don't dive into the list of ignored introductions)

Reply:
In this case (blocked contact or blocked node) the contact request won't be saved as an introduction at all. The question is if the remote contact should receive a follow rejection response or not in this case.